fw-child/template/social-networks.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+//
+// Display social network links.
+//
+
+?>
+
+<div class="social-networks">
+    <ul>
+        <?php
+
+            // Associative array of social networks field name suffix and Font Awesome icon class
+            $social_networks = [
+                'linkedin' => 'fa-linkedin',
+                'twitter' => 'fa-x-twitter',
+                'facebook' => 'fa-facebook',
+                'instagram' => 'fa-instagram',
+                'soundcloud' => 'fa-soundcloud'
+            ];
+
+            foreach ( $social_networks as $network => $icon ) {
+                $field_value = get_field( 'social_link_' . $network );
+
+                if ( !empty( $field_value ) ) {
+
+        ?>
+
+                    <li><a href="<?php echo $field_value; ?>" target="_blank" rel="noopener"><i class="fa-brands <?php echo $icon; ?>"></i></a></li>
+
+        <?php
+
+                }
```

Comment:
It could but I decided to make it this way so the editor sees the social network list in a clean way. If we do it the other way, the editor will see a broken social network icons list when one network is missing.